stdlib/scripts/check-docstrings.py

```diff
@@ -0,0 +1,33 @@
+# ===----------------------------------------------------------------------=== #
+# Copyright (c) 2024, Modular Inc. All rights reserved.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions:
+# https://llvm.org/LICENSE.txt
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ===----------------------------------------------------------------------=== #
+
+import subprocess
+from pathlib import Path
+import sys
+
+# TODO: Use the "mojo doc" directly when there is an option to
```

Comment:
**Suggestion** File a public GitHub issue and feel free to link [my comment](https://github.com/modularml/mojo/pull/2344#discussion_r1573831595) from the other PR where I suggested this feature.

Reply:
See https://github.com/modularml/mojo/issues/2447